src/Command/AddMassMediaCommand.php

```diff
@@ -22,11 +25,23 @@
  */
 class AddMassMediaCommand extends BaseCommand
 {
+    /**
+     * @var Registry|null
+     */
+    protected $doctrine;
```

Comment:
I think better will be inject `EntityManager` becouse:
```php
if (null !== $this->doctrine) {
    $this->doctrine->getManager()->getUnitOfWork()->clear();
}
```

Reply:
IMO, if we inject `EntityManager`, we will be leaving narrowing the scope only for the ORM, which I guess is not intended.

Reply:
Can you recheck it?
https://github.com/sonata-project/SonataMediaBundle/pull/1765#issuecomment-662569898

Reply:
I just see a reference and snippet there, but I really don't understand what are you trying to say.
Could you please explain?
Note that I'm just suggesting to use `$managerRegistry` as property name instead of `$doctrine`.

Reply:
Done. Can you recheck it? @phansys